docs/html-jsx.md

```diff
@@ -6,6 +6,6 @@ id: html-jsx
 <div class="jsxCompiler">
   <h1>HTML to JSX Compiler</h1>
   <div id="jsxCompiler"></div>
-  <script src="http://reactjs.github.io/react-magic/htmltojsx.min.js"></script>
+  <script src="https://reactjs.github.io/react-magic/htmltojsx.min.js"></script>
```

Comment:
In another PR `//` was used at first, but it was argued that the prefix should be `https//`: https://github.com/facebook/react/pull/3180#issuecomment-74913038

@zpao I am happy to change this, just let me know.